client/OPUNetTransportLayer.h

```diff
@@ -39,6 +39,13 @@ struct PeerInfo
 	PeerStatus status;
 	sockaddr_in address;
 	bool bReturnJoinPacket;
+
+	void Clear()
```

Comment:
Wasn't sure if we wanted to go ahead and move PeerInfo into its own header/cpp file and continue to expand its logic.

Reply:
I think it's fine to leave it where it is for now. It's meant to be a struct with all public fields, and doesn't really stand useful on it's own, but only in the context of where it is used.